toqito/state_opt/state_distinguishability.py

```diff
@@ -3,33 +3,38 @@
 import numpy as np
 import picos
 
-from toqito.matrix_ops import calculate_vector_matrix_dimension, vector_to_density_matrix
+from toqito.matrix_ops import calculate_vector_matrix_dimension, vector_to_density_matrix, vectors_to_gram_matrix
 from toqito.matrix_props import has_same_dimension
 
 
 def state_distinguishability(
     vectors: list[np.ndarray],
     probs: list[float] = None,
+    strategy: str = "min_error",
     solver: str = "cvxopt",
     primal_dual: str = "dual",
-) -> float:
+    **kwargs,
+) -> tuple[float, list[picos.HermitianVariable] | tuple[picos.HermitianVariable] | tuple[picos.RealVariable]]:
     r"""Compute probability of state distinguishability :cite:`Eldar_2003_SDPApproach`.
 
     The "quantum state distinguishability" problem involves a collection of :math:`n` quantum states
 
     .. math::
-        \rho = \{ \rho_0, \ldots, \rho_n \},
+        \rho = \{ \rho_1, \ldots, \rho_n \},
```

Comment:
Thanks for catching this. Instead of changing `rho_0` to `rho_1`, I think it might be easier to change `rho_n` to `rho_{n-1}`

Reply:
I think it might actually be better to do 1 -> n. Part of the reason for this is this would require making changes elsewhere in the docstring that I think would be cumbersome/confusing. 

I think this numbering scheme is distinct from any confusion surrounding the Python indexing, so I'd personally prefer 1 -> n

Reply:
Fine by me! I was trying to make it consistent with python indexing.